docs/src/references/child-devices.md

```diff
@@ -0,0 +1,62 @@
+# Child-Device Support
+
+Child-Device support allows to connect devices (e.g. sensors, actors, PLCs, any other kind of device) to the thin-edge device, to represent those devices in the cloud. Furthermore processes running on the thin-edge device can make use of child-device support to appear as a kind of a logical device in the cloud.
+
+## Terminology
+- **thin-edge device**:
+  the physical device thin-edge is running on
+- **external-device**:
+  some kind of physical device that is connected to the _thin-edge device_ (e.g. a PLC, a fieldbus device, a sensor or any other kind of device)
+- **child-device twin**:
+  the entity in the cloud, that represents the child-device; in case of C8Y that is a managed object
+- **child-id**:
+  a string uniquely referencing the _external-device_ and the corresponding _child-device twin_
+- **child-device agent**:
+  that piece of software that makes the glue between the _external-device_ and thin-edge APIs; could be located on the _external-device_ or running as process on the _thin-edge device_
+
```

Comment:
So far for the scope of that PR (Child-Device Support for C8Y Config Plugin) there is no need to introduce "external child device agent" and "local child device agent". However for the overall scope it might make sense. So I will consider that comment again in PR "requirements and use-cases" (#1195).